data/scripts/dependency-resolver.py

```diff
@@ -61,17 +61,17 @@ def handle_pkgconfig_check(args, conf):
     ver_match = True
 
     if exact_ver:
-        cmd = "pkg-config --exact-version=%s %s" % (exact_ver, pkg)
+        cmd = "%s --exact-version=%s %s" % (args.pkg_config, exact_ver, pkg)
```

Comment:
Unrelated to this patch, but creating a command string like this is most often wrong, as arguments are not properly escaped.

It would be a better idea to use an argument vector (akin to whatever you pass to any function in the `exec()` family), or, at the very least, pass the version and pkg-name in single quotes (which isn't 100% ideal but should avoid most gotchas).

Doing this will avoid problems if package names or versions contain special characters to the shell.